forward_engineering/alterScript/alterScriptHelpers/columnHelpers/defaultValueHelper.js

```diff
@@ -35,10 +36,12 @@ const getUpdatedDefaultColumnValueScriptDtos = ({ collection }) =>
 		})
 		.map(([columnName, jsonSchema]) => {
 			const newDefaultValue = jsonSchema.default;
+			const type = jsonSchema.mode || jsonSchema.childType || jsonSchema.type;
+			const isArrayType = Array.isArray(jsonSchema.array_type) && jsonSchema.array_type.length > 0;
 			const scriptGenerationConfig = {
 				tableName: getFullTableName(collection),
 				columnName: wrapInQuotes(columnName),
-				defaultValue: wrapInSingleQuotes({ name: newDefaultValue }),
+				defaultValue: decorateDefault(type, newDefaultValue, isArrayType),
```

Comment:
Aligned behavior with [convertColumnDefinition](https://github.com/hackolade/PostgreSQL/blob/ce0d5c397813bc2e78c1a1c5ea22ec40131bfa9f/forward_engineering/ddlProvider/ddlProvider.js#L223) - because not in all required cases, logic wraps values